isis/tests/ClipperPushBroomCameraTests.cpp

```diff
@@ -63,39 +67,43 @@ TEST_F(ClipperPbCube, ClipperPushBroomCameraWacTest) {
 
   EXPECT_PRED_FORMAT2(AssertQStringsEqual, cam->spacecraftNameLong(), "Europa Clipper");
   EXPECT_PRED_FORMAT2(AssertQStringsEqual, cam->spacecraftNameShort(), "Clipper");
   EXPECT_PRED_FORMAT2(AssertQStringsEqual, cam->instrumentNameLong(), "Europa Imaging System Push Broom Wide Angle Camera");
   EXPECT_PRED_FORMAT2(AssertQStringsEqual, cam->instrumentNameShort(), "EIS-PBWAC");
   EXPECT_PRED_FORMAT2(AssertQStringsEqual, cam->instrumentId(), "EIS-WAC-PB");
 
-  EXPECT_TRUE(cam->SetImage(1, 1));
-  EXPECT_NEAR(cam->UniversalLatitude(), 1.5088267433142744, 0.05);
-  EXPECT_NEAR(cam->UniversalLongitude(), 81.378861952389798, 0.05);
+  // Top
+  EXPECT_TRUE(cam->SetImage(2130, 30));
+  EXPECT_DOUBLE_EQ(cam->UniversalLatitude(),  -22.133032614015832);
+  EXPECT_DOUBLE_EQ(cam->UniversalLongitude(), 87.101796154127783);
   EXPECT_TRUE(cam->SetUniversalGround(cam->UniversalLatitude(), cam->UniversalLongitude()));
-  EXPECT_NEAR(cam->Sample(), 1, 0.0001);
-  EXPECT_NEAR(cam->Line(), 1, 0.05);
+  EXPECT_NEAR(cam->Sample(), 2130, 0.001);
+  EXPECT_NEAR(cam->Line(), 30, 0.0011);
 
-  EXPECT_TRUE(cam->SetImage(1, 2048));
-  EXPECT_NEAR(cam->UniversalLatitude(), 1.4965915498699245, 0.05);
-  EXPECT_NEAR(cam->UniversalLongitude(), 81.345482351767558, 0.05);
+  // Bottom
+  EXPECT_TRUE(cam->SetImage(2130, 2030));
+  EXPECT_DOUBLE_EQ(cam->UniversalLatitude(),  -22.016871728071468);
+  EXPECT_DOUBLE_EQ(cam->UniversalLongitude(), 86.674027874092516);
   EXPECT_TRUE(cam->SetUniversalGround(cam->UniversalLatitude(), cam->UniversalLongitude()));
-  EXPECT_NEAR(cam->Sample(), 1, 0.0001);
-  EXPECT_NEAR(cam->Line(), 2048, 0.05);
+  EXPECT_NEAR(cam->Sample(), 2130, 0.001);
+  EXPECT_NEAR(cam->Line(), 2030, 0.0017);
 
-  EXPECT_TRUE(cam->SetImage(4096, 2048));
-  EXPECT_NEAR(cam->UniversalLatitude(), -34.782957391682949, 0.05);
-  EXPECT_NEAR(cam->UniversalLongitude(), 68.590363919992924, 0.05);
+  // Right
+  EXPECT_TRUE(cam->SetImage(3580, 1024));
+  EXPECT_DOUBLE_EQ(cam->UniversalLatitude(), -72.765663853451784);
+  EXPECT_DOUBLE_EQ(cam->UniversalLongitude(), 108.87646404992245);
   EXPECT_TRUE(cam->SetUniversalGround(cam->UniversalLatitude(), cam->UniversalLongitude()));
-  EXPECT_NEAR(cam->Sample(), 4096, 0.0001);
-  EXPECT_NEAR(cam->Line(), 2048, 0.05);
+  EXPECT_NEAR(cam->Sample(), 3580, 0.001);
+  EXPECT_NEAR(cam->Line(), 1024, 0.037);
 
-  EXPECT_TRUE(cam->SetImage(4096, 1));
-  EXPECT_NEAR(cam->UniversalLatitude(), -34.782957391682949, 0.05);
-  EXPECT_NEAR(cam->UniversalLongitude(), 68.590363919992924, 0.05);
+  // Left
+  EXPECT_TRUE(cam->SetImage(544, 1024));
+  EXPECT_DOUBLE_EQ(cam->UniversalLatitude(), 20.853469071265028);
+  EXPECT_DOUBLE_EQ(cam->UniversalLongitude(), 113.21575238420702);
   EXPECT_TRUE(cam->SetUniversalGround(cam->UniversalLatitude(), cam->UniversalLongitude()));
-  EXPECT_NEAR(cam->Sample(), 4096, 0.0001);
-  EXPECT_NEAR(cam->Line(), 1, 0.05);
+  EXPECT_NEAR(cam->Sample(), 544, 0.001);
+  EXPECT_NEAR(cam->Line(), 1024, 0.034);
```

Comment:
The difference in Line for left and right of the wac image is 0.034 so not quite at a difference of 0.001